data/brands/amenity/bicycle_rental.json

```diff
@@ -1524,29 +1524,72 @@
       }
     },
     {
-      "displayName": "Santander Cycles",
+      "displayName": "Santander Cycles (Brunel)",
+      "locationSet": {"include": ["gb-eng"]},
+      "tags": {
+        "amenity": "bicycle_rental",
+        "brand": "Santander Cycles",
+        "brand:wikidata": "Q807961",
+        "brand:wikipedia": "en:Santander Cycles",
+        "network": "nextbike",
+        "operator": "nextbike",
+        "operator:wikidata": "Q2351279",
+        "operator:wikipedia": "en:nextbike"
+      }
+    },
+    {
+      "displayName": "Santander Cycles (London)",
       "id": "serco-b88dc4",
       "locationSet": {"include": ["gb-eng"]},
       "matchNames": [
         "barclays bikes",
         "barclays cycle hire",
         "boris bike",
-        "boris bikes",
-        "santander cycles mk"
```

Comment:
Despite looking through the history, I couldn't make sense of who added this line, but the MK one is operated by nextbike not Serco and isn't in the tfl network.